src/property.ts

```diff
@@ -43,13 +51,24 @@ type MafiaClasses =
   | Stat
   | Thrall;
 
+type MafiaClassProperty =
+  | FamiliarProperty
+  | LocationProperty
+  | MonsterProperty
+  | PhylumProperty
+  | StatProperty;
+
 const createMafiaClassPropertyGetter = <T extends MafiaClasses>(
   Type: typeof MafiaClass & (new () => T)
 ): ((property: string, default_?: T | null) => T | null) =>
   createPropertyGetter((value) => {
     if (value === "") return null;
-    const v = Type.get<T>(value);
-    return v === Type.get<T>("none") ? null : v;
+    try {
+      const v = Type.get<T>(value);
+      return v === Type.get<T>("none") ? null : v;
+    } catch {
```

Comment:
Will this lead to a Mafia briefly flashing red every time we get an error? I'm fine with that, of course, just curious

Reply:
What exception is thrown here? Can you narrow the catch expression?